column_mapping.go

```diff
@@ -52,6 +52,30 @@ func (group columnMappingGroup) lookup(path columnPath) leafColumn {
 	return leafColumn{columnIndex: -1}
 }
 
+func (group columnMappingGroup) lookupClosest(path columnPath) leafColumn {
+	for len(path) > 0 {
+		g, ok := group[path[0]].(columnMappingGroup)
+		if ok {
+			group, path = g, path[1:]
```

Comment:
does this break if you change the function signature to `(group *columnMappingGroup)`?

it's maybe fine it just makes me nervous if someone does that and doesn't notice this change.

Reply:
It does break, but then unit tests should break since the behavior of the method will change.